recipes/install-ua-harvester.rb

```diff
@@ -46,8 +48,10 @@
 AWS_DEFAULT_REGION="#{region}"
 MATTERHORN_REST_USER="#{rest_auth_info[:user]}"
 MATTERHORN_REST_PASS="#{rest_auth_info[:pass]}"
-MATTERHORN_HOST="#{engage_node[:private_ip]}"
-S3_LAST_ACTION_TS_BUCKET="#{stack_name}-ua-harvester"
+MATTERHORN_ENGAGE_HOST="#{engage_node[:private_ip]}"
```

Comment:
The `load_episodes` process needs to use pyhorn clients pointed at both engage and admin both.